### PR TITLE
fix: remove duplicated code

### PR DESCRIPTION
**Pull request by vicb**

### CODE REVIEW

File: packages/open-next/src/wrappers/aws-lambda-streaming.ts [hunk 1]
```diff
@@ -76,8 +76,6 @@ const handler: WrapperHandler = async (handler, converter) =>
 
       const streamCreator: StreamCreator = {
         writeHeaders: (_prelude) => {
-          _prelude.headers["content-encoding"] = contentEncoding;
```
Comment (conico974) [inline packages/open-next/src/wrappers/aws-lambda-streaming.ts:79]: Is it duplicated ?
I think we need to set it here, the compression is done by the wrapper itself

Comment (vicb) [reply packages/open-next/src/wrappers/aws-lambda-streaming.ts:79]: image

Check l84

Comment (conico974) [reply packages/open-next/src/wrappers/aws-lambda-streaming.ts:79]: :facepalm: My bad

**Review by conico974 — APPROVED** [review]
Thanks

**Pull request merged**